slowquant/qiskit_interface/interface.py

```diff
@@ -76,15 +70,17 @@ def __init__(  # pylint: disable=dangerous-default-value
         if ansatz_options is None:
             ansatz_options = {}
         allowed_ansatz = (
-            "tUCCSD",
-            "tPUCCD",
-            "tUCCD",
+            "fpUCCD",
+            "fUCCD",
             "tUPS",
             "fUCCSD",
             "QNP",
             "kSAfUpCCGSD",
-            "dUCCSD",
-            "kSAdUpCCGSD",
+            "SDSfUCCSD",
+            "kSASDSfUpCCGSD",
+            "fUCC",
+            "SDSfUCC",
+        "HF",
```

Comment:
I dont know how to make it indent via suggestion. But It would be nice to have HF on the same alignment:P